src/Patch/Cli/Command/ApplyCommand.php

```diff
@@ -95,6 +105,51 @@ protected function configure()
     }
 
     /**
+     * Override the current PHP version that's in use.
+     *
+     * @param $version
+     * @return $this
+     */
+    public function setPhpVersion($version)
+    {
+        $this->phpVersion = $version;
+        return $this;
+    }
+
+    /**
+     * Compiles a complete list of PHP version constraints from customer and combined packages
+     * and verifies if the current PHP version is compatiable.
+     *
+     * @param array $config
+     * @throws PhpVersionException
+     */
+    protected function checkPhpConstraint(array $config)
+    {
+        $versions = array();
+
+        if (isset($config['package']['php'])) {
+            $versions[$config['name']] = $config['package']['php'];
+        }
+
+        if (isset($config['combined'])) {
+            foreach ($config['combined'] as $combinedPackage => $combinedConfig) {
+                if (!isset($combinedConfig['package']['php'])) {
+                    continue;
```

Comment:
Minor code style thing: Perhaps change to `if (isset( ...` and set the version in the if block to avoid the `continue`?